app/controllers/pages_controller.rb

```diff
@@ -2,5 +2,6 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [ :home ]
 
   def home
+    redirect_to missions_path
```

Comment:
Why do you redirect to missions index? shouldn't it be campaigns index?

Reply:
My bad. I update that